Bottom: 0cff76fce94b300ea6551c6b6ca415a2b7c0cbce
Top:    a4f7defaa98d26a5ee51742c4aa0819530e51afd
Author: Ian Kent <ikent@redhat.com>
Date:   2013-10-02 09:30:30 +0800

Refresh of autofs-5.0.7-add-std-vars-to-program-map-invocation.patch

---

diff --git a/lib/mounts.c b/lib/mounts.c
index ed92b43..f666f29 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -311,7 +311,7 @@ struct substvar *addstdenv(struct substvar *sv)
 
 	tsv = pthread_getspecific(key_thread_stdenv_vars);
 	if (tsv) {
-		struct substvar *mv;
+		const struct substvar *mv;
 		int ret;
 		long num;
